[16.0][web_responsive] [FIX] Split button templates to show translated labels

hugosantosred opened the pull request:
Fixes #2439

Edit: I've mispelled the issue number

hugosantosred committed:
[FIX] Split button templates to show translated labels

Fixes #2439

bvanjeelharia commented on web_responsive/static/src/legacy/xml/form_buttons.xml:
```diff
@@ -9,9 +9,17 @@
+    <t t-name="web_responsive.icon_button_create" owl="1">
         <i t-attf-class="fa fa-#{icon}" t-att-title="label" />
-        <span class="d-none d-sm-inline" t-esc="label" />
+        <span class="d-none d-sm-inline"> Create</span>
```
Can you please change this to New as the newer update of odoo 16 has it as New instead of Create?

hugosantosred replied:
Done! thanks for the review

Tardo commented on web_responsive/static/src/legacy/xml/form_buttons.xml:
```diff
@@ -9,9 +9,17 @@
     <!-- Template for buttons that display only the icon in xs -->
-    <t t-name="web_responsive.icon_button" owl="1">
+    <t t-name="web_responsive.icon_button_create" owl="1">
         <i t-attf-class="fa fa-#{icon}" t-att-title="label" />
```
```suggestion
        <i t-attf-class="fa fa-#{icon}" title="New" />
```

Tardo commented on web_responsive/static/src/legacy/xml/form_buttons.xml:
```diff
@@ -9,9 +9,17 @@
+        <span class="d-none d-sm-inline"> New</span>
+    </t>
+    <t t-name="web_responsive.icon_button_save" owl="1">
+        <i t-attf-class="fa fa-#{icon}" t-att-title="label" />
```
```suggestion
        <i t-attf-class="fa fa-#{icon}" title="Save" />
```

Tardo commented on web_responsive/static/src/legacy/xml/form_buttons.xml:
```diff
@@ -9,9 +9,17 @@
+        <span class="d-none d-sm-inline"> Save</span>
+    </t>
+    <t t-name="web_responsive.icon_button_discard" owl="1">
+        <i t-attf-class="fa fa-#{icon}" t-att-title="label" />
```
```suggestion
        <i t-attf-class="fa fa-#{icon}" title="Discard" />
```

Tardo commented on web_responsive/static/src/legacy/xml/form_buttons.xml:
```diff
@@ -36,9 +44,8 @@
             >
-                <t t-call="web_responsive.icon_button">
+                <t t-call="web_responsive.icon_button_create">
                     <t t-set="icon" t-value="'plus'" />
```
This can go directly into the template no? I don't see when someone would want a different icon for the same action.

Tardo reviewed (changes requested):
Thanks for the work! Only code review...

hugosantosred commented:
@Tardo Changes done. You're right the icon should be the same for actions of the same type.  So I've simplified the templates to have the icon already set, and fixed the titles on them.

Thanks for the review

Tardo reviewed (approved):
LGTM, thanks!

remi-filament reviewed (approved):
Tested OK on runboat in both English and French. Thanks !

pedrobaeza commented:
/ocabot merge minor

pedrobaeza commented:
/ocabot merge minor
